src/ps/psreaddata.cpp

```diff
@@ -502,13 +516,22 @@ PetscErrorCode PSReadMatPowerData(PS ps, const char netfile[]) {
           -1; /* xx_end_line points to the next line after the record ends */
   PetscInt gen_start_line = -1, gen_end_line = -1;
   PetscInt br_start_line = -1, br_end_line = -1;
+  PetscInt dcline_start_line = -1, dcline_end_line = -1;
   PetscInt gencost_start_line = -1, gencost_end_line = -1;
   PetscInt genfuel_start_line = -1, genfuel_end_line = -1;
   PetscInt loadcost_start_line = -1, loadcost_end_line = -1;
+
+  /* Number of lines for each data */
+  PetscInt bus_numlines, gen_numlines, br_numlines, dcline_numlines;
+  PetscInt load_numlines = 0;
```

Comment:
No sure these variables are used anywhere of substance

Reply:
These are used for counting the number of buses, gens, branches, dclines, and loads.